protocol/avdl/chat1/local.avdl

```diff
@@ -94,20 +94,23 @@ protocol local {
   }
 
   record MessageInfoLocal {
-    boolean isNew;
     string senderUsername;
     string senderDeviceName;
   }
 
-  record Message {
+  record MessageFromServerUnboxed {
     MessageServerHeader serverHeader;
     MessagePlaintext messagePlaintext;
+  }
 
+  record MessageFromServerUnboxedWithContext {
```

Comment:
Aha that's a good point. I don't think `MessageFromServerUnboxed` is used anywhere outside the service at all. So I think I'll just remote `MessageFromServerUnboxed` from avdl and rename `MessageFromServerUnboxedWithContext` into `MessageFromServerUnboxed`!